src/main/java/edu/hawaii/its/api/wrapper/UpdateTimestampCommand.java

```diff
@@ -40,6 +59,14 @@ public UpdatedTimestampResult execute() {
         return new UpdatedTimestampResult(wsAssignAttributesResults);
     }
 
+    public List<UpdatedTimestampResult> execute2() {
+        List<UpdatedTimestampResult> updatedTimestampResults = new ArrayList<>();
+        WsAssignAttributesResults wsAssignAttributesResults = this.gcAssignAttributes.execute();
+        for (int i = 0; i < wsAssignAttributesResults.getWsGroups().length; i++) {
+            updatedTimestampResults.add(new UpdatedTimestampResult(wsAssignAttributesResults, i));
+        }
+        return updatedTimestampResults;
+    }
```

Comment:
I think it would ideal if we could have the same execute() function that works on both UpdateTimestampCommands created with a groupPath and a list of groupPaths

Reply:
I think if we wanted one "execute" method to handle both a single groupPath and a list of groupPaths, it would probably have to return a list of size equal to the number of paths we want to update. The problem with this is that the UpdateTImestampCommand implements the Command interface and that method signature for execute is that it returns type T. A work around would be make it so it doesn't implement the Command interface, then we could make the execute method return a list or we could make the Command interface's execute method return a List<T>, but everything else that implements it would need to be updated to deal with execute returning a list instead of an object.